public/js/bundle.js

```diff
@@ -101,6 +101,7 @@ class TableView {
 	initDomReferences() {
 		this.headerRowEl = document.querySelector('THEAD TR');
 		this.sheetBodyEl = document.querySelector('TBODY');
+		this.sheetFootEl = document.querySelector('TFOOT TR');
```

Comment:
Table terminology: footer. This part is very similar to the "this.headerRowEl", so it's more consistent to name it "this.footerRowEl". Aim for consistency and patterns everywhere you can.